shared/src/packages/trucks/libs/enums/truck-capacity.enum.ts

```diff
@@ -1,6 +1,6 @@
 const TruckCapacity = {
   MIN: 1,
-  MAX: 4,
+  MAX: 999,
```

Comment:
why? 🥲

Reply:
The QA team said that the capacity should be 1-999, and they even created a separate bug ticket for this. I can revert it back, it will take a minute))

Reply:
If QAs said so... 🥲